src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -72,6 +72,8 @@ public CosmosDBTriggerAttributeBindingProvider(INameResolver nameResolver, Docum
                     CollectionName = ResolveAttributeValue(attribute.CollectionName)
                 };
 
+                documentCollectionLocation.ConnectionPolicy.UserAgentSuffix = "CosmosDBTrigger";
```

Comment:
nit: Since you're using this twice, make it a constant? This is for you guys to be able to get telemetry, right? I'll leave the header name up to you then :-)

Reply:
Yup, it's so we can track the useragent :smile: